akka-http-core/src/main/scala/akka/http/impl/settings/ServerSettingsImpl.scala

```diff
@@ -51,6 +53,7 @@ object ServerSettingsImpl extends SettingsCompanion[ServerSettingsImpl]("akka.ht
   implicit def timeoutsShortcut(s: js.ServerSettings): js.ServerSettings.Timeouts = s.getTimeouts
 
   /** INTERNAL API */
+  @InternalApi
```

Comment:
In order to do this, and not having MiMa complain, it would be cool to have https://github.com/akka/akka-http/pull/871 merged in :)